nikola/data/samplesite/pages/quickstart.rst

```diff
@@ -22,7 +22,7 @@ quick reference`_ document.
 __
 .. _Quick reStructuredText: ../quickref/
 .. _master quick reference:
-   http://docutils.sourceforge.net/docs/user/rst/quickref.html
```

Comment:
`quickstart.rst` is a third-party document. There’s no need to change those links, and if we ever update this file with a new upstream version, those links will go back to `http` anyway.

Reply:
I've had a look around but haven't been able to find the original reST version of that document in the docutils repository. Would you happen to know where it can be found?

Reply:
Both quickstart and quickref live here: https://sourceforge.net/p/docutils/code/HEAD/tree/trunk/docutils/docs/user/rst/

Reply:
The quickstart document isn't in reST though. How did Nikola arrive at an reST version?

Reply:
`quickref.rst` is basically one big `.. raw:: html` block. The `.html` extension was added to the default POSTS/PAGES lists [in 2016](https://github.com/getnikola/nikola/commit/09a9092143115d8153afc3e666a6edb20afd12f1), and `quickref.rst` was [added in 2012](https://github.com/getnikola/nikola/commit/42f86fd5e8a05d418282ffd0054dc75efe21ecab). Changing the extension might be possible now, although I’m not sure about any side-effects this change could bring (importers might not add `.html` to POSTS/PAGES).